controllers/mongodb_tls_test.go

```diff
@@ -23,10 +23,10 @@ func TestStatefulSet_IsCorrectlyConfiguredWithTLS(t *testing.T) {
 	mdb := newTestReplicaSetWithTLS()
 	mgr := client.NewManager(&mdb)
 
-	client := mdbClient.NewClient(mgr.GetClient())
-	err := createTLSSecret(client, mdb, "CERT", "KEY", "")
+	cli := mdbClient.NewClient(mgr.GetClient())
```

Comment:
client is shadowing package `client`, e.g. used in line 24. I was getting warnings about it. So either we alias package or change var name. Yes `cli` is not a great name :(

Reply:
@irajdeep I mistakenly deleted your Q: why `cli` because `client` was more descriptive